components/kyma-environment-broker/internal/orchestration/handlers/cancel_test.go

```diff
@@ -0,0 +1,88 @@
+package handlers
+
+import (
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/orchestration"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	fixOrchestrationID = "test-id"
+)
+
+func TestNewCanceler(t *testing.T) {
```

Comment:
You don't test NewCanceler constructor in this code block. Consider changing this to more descriptive approach, like:
```golang
func TestCanceler_Cancel(t *testing.T) {
	t.Run("should cancel existing orchestration" , func(t *testing.T) {
		/* 
		...
		 */
	})
	t.Run("should return error when orchestration not found" , func(t *testing.T) {
		/*
			...
		*/
	})
}

func TestCanceler_CancelForID(t *testing.T) {
	t.Run("should cancel orchestration" , func(t *testing.T) {
		/*
			...
		*/
	})
	t.Run("should return error when orchestration not found" , func(t *testing.T) {
		/*
			...
		*/
	})
}
```